pyproject.toml

```diff
@@ -84,6 +84,7 @@ filterwarnings = [
   "ignore:pythonw executable not found:UserWarning:",
   "ignore:data shape .* exceeds GL_MAX_TEXTURE_SIZE:UserWarning",
   "ignore:For best performance with Dask arrays in napari:UserWarning:",
+  "ignore:The square brush shape is deprecated and will be removed:FutureWarning",
```

Comment:
this is fine too, but if it's just one or two tests that are failing because of this warning, you might also consider using a `with pytest.warngs(FutureWarning):` context right before the call that creates the warning.  (That would also remind us to remove the ignore it when the warning goes away eventually)